src/test/lambda/explorer/cloudFormationNodes.test.ts

```diff
@@ -41,275 +52,237 @@ describe('CloudFormationStackNode', () => {
 
-            public async deleteFunction(name: string): Promise<void> {}
+    it('has child nodes with CloudFormation contextValue', async () => {
+        const childNodes = await testNode.getChildren()
 
-            public async invoke(name: string, payload?: Lambda._Blob): Promise<Lambda.InvocationResponse> {
-                return ({} as any) as Lambda.InvocationResponse
-            }
+        childNodes.forEach(node =>
+            assert.strictEqual(
+                node.contextValue,
+                CONTEXT_VALUE_CLOUDFORMATION_LAMBDA_FUNCTION,
+                'expected the node to have a CloudFormation contextValue'
+            )
+        )
+    })
 
-            public async *listFunctions(): AsyncIterableIterator<Lambda.FunctionConfiguration> {
-                yield* this.lambdas
-            }
+    it('only includes functions which are in a CloudFormation stack', async () => {
+        lambdaFunctionNames = ['lambda1', 'lambda2', 'lambda3']
+        cloudFormationStacklambdaFunctionNames = ['lambda1', 'lambda3']
 
-            public addLambdaResource(name: string): void {
-                this.lambdas.push(({
-                    FunctionName: name
-                } as any) as Lambda.FunctionConfiguration)
-            }
-        }
+        const childNodes = await testNode.getChildren()
+        assert.strictEqual(
+            childNodes.length,
+            cloudFormationStacklambdaFunctionNames.length,
+            'Unexpected child node count'
+        )
 
-        const lambda1Name = 'lambda1Name'
-        const lambda2Name = 'lambda2Name'
-        const lambda3Name = 'lambda3Name'
+        assert.deepStrictEqual(
+            new Set<string>(childNodes.map(node => node.label!)),
+            new Set<string>(cloudFormationStacklambdaFunctionNames),
+            'Unexpected child sort order'
+        )
+    })
 
-        const cloudFormationClient = new TestMockCloudFormationClient('code')
-        cloudFormationClient.addLambdaResource(lambda1Name)
-        cloudFormationClient.addLambdaResource(lambda3Name)
+    // TODO : CloudFormation Stack Nodes don't sort their child nodes
+    // it('sorts child nodes', async () => {
+    //     lambdaFunctionNames = UNSORTED_TEXT
+    //     cloudFormationStacklambdaFunctionNames = UNSORTED_TEXT
 
-        const lambdaClient = new MockLambdaClient('code')
-        lambdaClient.addLambdaResource(lambda1Name)
-        lambdaClient.addLambdaResource(lambda2Name)
-        lambdaClient.addLambdaResource(lambda3Name)
+    //     const childNodes = await testNode.getChildren()
 
-        // TODO: Move to MockToolkitClientBuilder
-        ext.toolkitClientBuilder = {
-            createCloudFormationClient(regionCode: string): CloudFormationClient {
-                return cloudFormationClient
-            },
+    //     const actualChildOrder = childNodes.map(node => node.label)
+    //     assert.deepStrictEqual(actualChildOrder, SORTED_TEXT, 'Unexpected child sort order')
+    // })
```

Comment:
Offline conversation confirmed that this is a good test for functionality that doesn't exist. Please create an issue and add it to the comment so we remember